omg/entities/elements.py

```diff
@@ -20,16 +20,8 @@ def __setattr__(self, attr, value):
         self[attr] = value
 
 
-# Recursively convert the dictionary and all its nested dictionaries to DotDict
-def to_dotdict(d: dict) -> DotDict:
-    """Convert dictionary and its nested ones to DotDict."""
-    if isinstance(d, dict):
-        return DotDict({k: to_dotdict(v) for k, v in d.items()})
-    return d
-
-
 with open(ELEMENTS_JSON_DIR, "r") as file:
     ELEMENTS: Dict[str, Dict] = json.load(file)
 
 # Convert nested dictionaries to DotDict instances
-ELEMENTS = to_dotdict(ELEMENTS)
+ELEMENTS: DotDict[str, DotDict] = DotDict(ELEMENTS)
```

Comment:
I don't understand why you removed this `to_dotdict()` method and used directly the constructor call is a default dict() constructor as you haven't overloaded it. It seemed to be that `to_dotdict()` was creating the DotDict with the right format you wanted.

Reply:
Both `DotDict` and `to_dotdict()` performs recursive call of `DotDict`, so both results in the same format of dictionary. 
Therefore, `to_dotdict()` was unnecessary.

Reply:
I don't think there is a recursive call of DotDict. Do you mean `if isinstance(value, dict): value = DotDict(value)` as recursive? If so, it is not. As it just modifies the output value whenever someone tries to get that from the dictionary. It does not replace the value in the dictionary, meaning that everytime one calls the get method with the same value, value = DotDict(value) would run, which is redundant.